docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -233,408 +122,121 @@ is used.
-    - ``false``
-
-Parquet format configuration
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-The following properties are used to configure the read and write operations
-with Parquet files performed by the Iceberg connector.
-
-.. list-table:: Parquet format configuration properties
-    :widths: 30, 50, 20
-    :header-rows: 1
-
-    * - Property Name
-      - Description
-      - Default
-    * - ``parquet.max-read-block-row-count``
-      - Sets the maximum number of rows read in a batch.
-      - ``8192``
-    * - ``parquet.optimized-reader.enabled``
-      - Whether batched column readers should be used when reading Parquet files
-        for improved performance. Set this property to ``false`` to disable the
-        optimized parquet reader by default. The equivalent catalog session
-        property is ``parquet_optimized_reader_enabled``.
-      - ``true``
-    * - ``parquet.optimized-nested-reader.enabled``
-      - Whether batched column readers should be used when reading ARRAY, MAP
-        and ROW types from Parquet files for improved performance. Set this
-        property to ``false`` to disable the optimized parquet reader by default
-        for structural data types. The equivalent catalog session property is
-        ``parquet_optimized_nested_reader_enabled``.
-      - ``true``
-
-.. _iceberg-authorization:
-
-Authorization checks
-^^^^^^^^^^^^^^^^^^^^
-
-You can enable authorization checks for the connector by setting
-the ``iceberg.security`` property in the catalog properties file. This
-property must be one of the following values:
+Metastores
+----------
 
-.. list-table:: Iceberg security values
-  :widths: 30, 60
-  :header-rows: 1
+Iceberg natively uses the concept of a catalog. The subsection titles in this
+section refer to the type of Iceberg metadata catalog. The examples in each
+subsection depict th contents of a Trino catalog file that has an Iceberg
+metadata catalog as its source.
 
-  * - Property value
-    - Description
-  * - ``ALLOW_ALL``
-    - No authorization checks are enforced.
-  * - ``SYSTEM``
-    - The connector relies on system-level access control.
-  * - ``READ_ONLY``
-    - Operations that read data or metadata, such as :doc:`/sql/select` are
-      permitted. No operations that write data or metadata, such as
-      :doc:`/sql/create-table`, :doc:`/sql/insert`, or :doc:`/sql/delete` are
-      allowed.
-  * - ``FILE``
-    - Authorization checks are enforced using a catalog-level access control
-      configuration file whose path is specified in the ``security.config-file``
-      catalog configuration property. See
-      :ref:`catalog-file-based-access-control` for information on the
-      authorization configuration file.
+The connector supports multiple Iceberg catalog types; you may use
+either a Hive metastore service (HMS), AWS Glue, or a REST catalog. The catalog
+type is determined by the ``iceberg.catalog.type`` property. It can be set to
```

Comment:
Maybe the values should be in each section ..

Reply:
That has to be done in a separate PR as it introduces changes to existing technical content.